### ISSUE TITLE
Allow background color customization

### ISSUE THREAD

**Issue by gabrielbidula** [TablePlus/TablePlus#1581]
I would like to change the full black background color for something softer.
Is it possible?

cheers

**Comment by tony-develops** [comment]
^

I agree. There is a lot of customisation with the text, however there is only two different background you can choose from. It would be nice to have the ability to choose specific background so it matches with the theme.

**Comment by adamryan-grin** [comment]
Agreed... I am just switching over to TablePlus and this is a major bummer. I like to use solarized colors and the fact that you can't manage themes in general is annoying but not being able to change the background color is really surprising

**Comment by gabrielbidula** [comment]
No update on this? 👎

**Comment by apsylone** [comment]
👍
